src/js/api/errors.js

```diff
@@ -29,6 +29,16 @@ export const SETUP_ERROR_LICENSE_EXPIRED = 100013;
  */
 export const SETUP_ERROR_LOADING_CORE_JS = 101000;
 
+/**
+ * @enum {ErrorCode} Setup failed because the playlist failed to load
+ */
+export const SETUP_ERROR_LOADING_PLAYLIST = 102000;
+
+/**
+ * @enum {ErrorCode} Playback stopped because the playlist failed to load
+ */
+export const ERROR_LOADING_PLAYLIST = 203000;
```

Comment:
should be code 202000. The only difference between SETUP_ERROR_LOADING_PLAYLIST and ERROR_LOADING_PLAYLIST should be the first digit